macros/union_data.sql

```diff
@@ -92,16 +92,38 @@
     {%- endif -%}
 
 {%- else -%}
-    {%- set relation=adapter.get_relation(
-        database=var(database_variable, default_database),
-        schema=var(schema_variable, default_schema),
-        identifier=var(default_schema ~ '_' ~ table_identifier ~ '_' ~ 'identifier', table_identifier)) -%}
-
-{%- set table_exists=relation is not none -%}
+    {% set exception_schemas = {"linkedin_company_pages": "linkedin_pages", "instagram_business_pages": "instagram_business"} %}
```

Comment:
are these the only `exception_schemas` across ALL of our packages? or just those that use `union_data`? if it's the latter, we'll need to remember to check this for any packages we add unioning-ability too. 

wondering again about maintaining a spreadsheet (or seed file in fivetran_utils?) with info like this on each package

Reply:
I only checked for the packages that currently leverage the `union_data` macro. I agree that this will be something we will need to take into consideration prior to using the macro in a package.

To also note, if a package is an exception (like the two listed here) it will fail. So we will be able to catch it pretty easily when implementing the macro.